pkgs/jni/pubspec.yaml

```diff
@@ -30,7 +30,7 @@ dependencies:
 dev_dependencies:
   dart_flutter_team_lints: ^3.2.0
   ## Pin ffigen version because we are depending on internal APIs.
-  ffigen: 8.0.2
+  ffigen: 16.1.0
```

Comment:
This should probably be a path dependency. In the future with a workspace, any internal API changes will need to be changed in the same commit.

Reply:
I kept this pinned and added an dep override not to break @mosuem's workflow.